test/loader/error-test/loader-error.test.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+// eslint-disable-next-line node/no-unpublished-require
+const { run } = require('../../utils/test-utils');
+
+describe('loader error regression test for #1581', () => {
+    it(`should not ignore loader's error produce a failing build`, () => {
+        // Ignoring assertion on stderr because ts-loader is producing depreciation warnings
+        // with webpack@v5.0.0-beta.24 -> https://github.com/TypeStrong/ts-loader/issues/1169
+        const { stdout } = run(__dirname, [], false);
+
+        expect(stdout).toContain('[1 error]');
+        expect(stdout).toContain(`Cannot assign to 'foobar' because it is a constant`);
```

Comment:
Can we add test on errorCode?

Reply:
Any suggestion on how can we drive webpack's error code here?

Reply:
execa should exit with same code, just check `code` from execa, it should be not `0`

Reply:
It is `0` 😞 .

Reply:
It is bug, I think it https://github.com/webpack/webpack-cli/pull/1739 should fix it, so our process will have original exit code

Reply:
Maybe you can help @anshumanv with this PR